raster/src/main/scala/geotrellis/raster/io/geotiff/MultiBandGeoTiff.scala

```diff
@@ -37,8 +37,14 @@ class MultiBandGeoTiff(
     new ArrayMultiBandTile(newBands)
   }
 
-  def subset(bands: Int*)(implicit d: DummyImplicit): ArrayMultiBandTile =
-    subset(bands)
+  def bands(bandSequence: Int*)(implicit d: DummyImplicit): ArrayMultiBandTile =
+    bands(bandSequence)
+
+  def subset(bandSequence: Seq[Int]): ArrayMultiBandTile =
+    bands(bandSequence.sorted)
+
+  def subset(bandSequence: Int*)(implicit d: DummyImplicit): ArrayMultiBandTile =
```

Comment:
you know what, I think `bands` is enough, I don't think we need a `subset` method.